src/common/com/intellij/plugins/haxe/ide/module/HaxeModuleBuilder.java

```diff
@@ -51,6 +74,75 @@ public void moduleCreated(@NotNull Module module) {
     final CompilerModuleExtension model = (CompilerModuleExtension)CompilerModuleExtension.getInstance(module).getModifiableModel(true);
     model.setCompilerOutputPath(model.getCompilerOutputUrl());
     model.inheritCompilerOutputPath(false);
+
+    final Project project = module.getProject();
+    List<Pair<String, String>> sourcePaths = getSourcePaths();
+    String srcPath = null;
+
+    if ((sourcePaths.size() > 0)) {
+      srcPath = sourcePaths.get(0).getFirst();
```

Comment:
I'm not so thrilled about just adding it to the first source path.  In fact, I'm not sure that we should be generating a file at all.  I would think that adding a main method to an existing class would be enough -- and the more useful -- help.  (And, in reality, I thought you were only going to put up a warning dialog.)